src/core/mount.js

```diff
@@ -37,6 +38,24 @@ function mount(context) {
                 // Logging should be acceptable in an error handler.
                 // eslint-disable-next-line no-console
                 console.error(error, info);
+
+                const { filename, lineNumber, column } = extractErrorDetails(
+                  info.componentStack
+                );
+
+                // Call window.onerror to send the error to the error logging system.
+                if (window.onerror) {
+                  window.onerror(
+                    `${error.name}: ${error.message}`,
+                    filename,
+                    lineNumber,
+                    column,
+                    {
+                      ...info,
+                      stack: info.componentStack
+                    }
+                  );
+                }
```

Comment:
Great that you found a way to forward the errors. I am excited to see it in action!
I think it would be worth commenting here why it is necessary to do so.
And that we could be the whole forwarding block in an external utils function called something like: `forwardError`. In order to reduce clutter in the mount.js file.